src/components/Stage.ts

```diff
@@ -309,6 +309,11 @@ export class Stage extends Container {
     }
   }
 
+  public load(html: string, clear: boolean = true): Stage {
```

Comment:
Without this overloading load will return this as `Container` instead of `Stage`.

Reply:
How about do a casting while using? i.e.

stage: Stage =...;
(stage.load(html) as Stage).callAStageFunction();

The casting will be ignored while compiling to JS, but the function override will not.